pkg/kubectl/cmd/util/editor/editoptions.go

```diff
@@ -440,6 +442,7 @@ func (o *EditOptions) annotationPatch(update *resource.Info) error {
 	return nil
 }
 
+// GetApplyPatch returns original content, a merge patch and a patch type.
```

Comment:
It might be a good idea to use named result parameters here. I think this is actually one of the example used in [Go Code Review Comments](https://github.com/golang/go/wiki/CodeReviewComments#named-result-parameters)

Reply:
Since you can describe the returned values using the named result parameters, I'd be interested to know what this function does.